### PR TITLE
chore(deps): upgrade all deps

### LINKED ISSUE #154: Move to webpack 3
Now that webpack 3 [is out](medium.com/webpack/webpack-3-official-release-15fd2dd8f07b), we should use it, use the new features of it. Along with maybe the autodll plugin. @iam4x has a good experience on this so maybe he could help.

The goal is not to be left behind in term of build tools.

### PR DESCRIPTION
**Pull request by vvo**
+ remove babili, was minifying the actual output inside the npm package, thus making source file errors complex to read for consumers
+ upgrade to webpack 3
+ fix an old require to storybook addons https://github.com/storybooks/storybook/issues/1480

Linked to #162
fixes #154

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,39 +30,39 @@
...
     "babel-jest": "^20.0.3",
-    "babel-loader": "^7.0.0",
+    "babel-loader": "^7.1.1",
     "babel-plugin-inline-json-import": "^0.2.1",
     "babel-plugin-lodash": "^3.2.11",
     "babel-plugin-transform-react-jsx-source": "^6.22.0",
     "babel-polyfill": "^6.23.0",
-    "babel-preset-babili": "^0.0.12",
-    "babel-preset-env": "^1.5.2",
+    "babel-preset-babili": "^0.1.4",
```
Comment (Haroenv) [inline package.json]: babili should be removed here then I guess

Comment (vvo) [inline package.json]: ththx

**Comment by vvo** [comment]
Maybe @Haroenv You want to have another look. Should be good now

File: .babelrc [hunk 2]
```diff
@@ -1,7 +1,18 @@
+// This could be written a lot more easily using .babelrc.js but currently
+// not released. And the merging around the env option is broken
+// refs: https://github.com/babel/babel/issues/5276
```
Comment (Haroenv) [inline .babelrc]: sad

File: .babelrc [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
 {
+  "presets": [
```
Comment (Haroenv) [inline .babelrc]: so Jest now uses this?

Comment (vvo) [inline .babelrc]: yes

**Comment by vvo** [comment]
Good to 🚢

**Comment by Haroenv** [comment]
Weird that the placeholder text is slightly darker now, even though we didn't change css. Is that an Argos bug @oliviertassinari

**Comment by oliviertassinari** [comment]
@Haroenv The best way to be sure it to check out the storybook locally.
Given that CSS related dependencies were updated, it could be prefixing behavior that changed.

**Comment by oliviertassinari** [comment]
From what I can see, we have 4 builds that are failing, the new output is consistent across 100% of the failing builds. That looks like a regression. To be confirmed though.

**Comment by vvo** [comment]
Maybe just an update on some CSS, that one is OK

**Pull request merged**